internal/provider/service.go

```diff
@@ -29,7 +29,7 @@ func service() *schema.Resource {
 			"api_secret":                                       {Type: schema.TypeString, Computed: true},
 			"clients_per_developer":                            {Type: schema.TypeInt, Required: false, Optional: true},
 			"client_id_alias_enabled":                          {Type: schema.TypeBool, Required: false, Optional: true, Default: false},
-			"attribute":                                        createAttributeSchema(),
+			"attributes":                                       createAttributeSchema(),
```

Comment:
I think we should not break the schema backwards compatibility

Reply:
Yeah. That is a good point. It could cause issues.
@hefa  and I discussed this and agreed to change the schema name. [As it is defined in our API documentation](https://docs.authlete.com/en/shared/latest#get-/api/service/get/list), it is "attributes". We thought having consistency across API documentation and terraform provider would be a good idea. 
This change will bring backward compatibility issues. But I think we still have a chance to change if needed, before many people start using our TF provider.